templates/incl/editor-and-output.html

```diff
@@ -25,7 +25,8 @@
         <!-- The actual editor -->
         <div id="editor" style="background: #272822; font-size:0.95em; color: white; direction: {{ g.dir }};" lang="en" blurred='false'
           {% if editor_readonly %}data-readonly="true"{% endif %}
-          class="w-full flex-1 text-lg rounded min-h-0 overflow-hidden fixed-editor-height"></div>
+          class="w-full flex-1 text-lg rounded min-h-0 overflow-hidden fixed-editor-height"
+             data-kwlang="{{ current_language().lang }}" data-level="{{ level }}"></div>
```

Comment:
Same here, if we can put this attribute on a `<div>` that contains all editors, it should work already?